### DISCUSSION TITLE
32k模型调用问题

### DISCUSSION THREAD

**Discussion by SXxinxiaosong** [zai-org/ChatGLM3#489] [Q&A] [answered]
metadata, content = response.split("\n", maxsplit=1)
ValueError: not enough values to unpack (expected 2, got 1)
 请问大家调用32k的模型时遇到这种问题是怎么解决的呢

**Comment by zRzRzRzRzRzRzR** [comment]
这个问题在huggingface中有人提出，我们要查看一下

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
6b模型已经修理，去6b的模型model card查看一下新更新的内容
